packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx

```diff
@@ -202,7 +203,28 @@ const PayablesTableBase = ({
   const areCounterpartsLoading = useAreCounterpartsLoading(payables?.data);
   const dateFormat = useDateFormat();
 
-  const columns = useMemo<GridColDef[]>(() => {
+  const defaultFieldOrder = useMemo<string[]>(
+    () => [
+      'document_id',
+      'counterpart_id',
+      'created_at',
+      'issued_at',
+      'due_date',
+      'status',
+      'amount',
+      'pay',
+    ],
+    []
+  );
```

Comment:
Is there a unique use case for utilising `useMemo`  here? I see this is an inexpensive operation that doesn't require optimisation/caching by this hook. 

It is a constant that doesn't run any logic or depend on any variable within the component; it can even be moved outside the component. Depending on the folder structure or reusability, it can be exported/imported externally or just moved outside the component within the same file.

Reply:
This useMemo was recommended by eslint

<img width="1108" alt="Screenshot 2024-09-20 at 15 45 09" src="https://github.com/user-attachments/assets/fb6cc76f-5c66-40b8-a6a3-8094193332b8">

Reply:
Thanks, completely forgot we can avoid this complexity by moving out as a const